src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx

```diff
@@ -0,0 +1,115 @@
+  * Understand what a feature is used for, and the benefits of the feature beyond the existing heading or other content. 
+  * Perform specific tasks.
+  * Understand what to do next. Are there recommended follow-on actions? The Apdex troubleshooting onramp provides a nice example for this. It tells users: "Compare the Apdex chart with other charts (like Web transaction time and Error rate) to look for the causes for the Apdex change."
+
+## FAQs
+
+### When is onramps content needed?
+
+Not every section of the UI requires an onramp. Use them primarily at a high level on the page. If the design calls for an onramp in a spot where you don't have much to say, then talk to the designer and/or PM for the section about modifying the design or removing the onramp.
+
+### How should I name the onramp titles?
+
+Titles (headers) can be any format, as long as they're short/succinct. For example, you can use an FAQ format where the headers are in question format. Here's the **Issues** tile onramp with titles formatted into questions.
+
+<img
+    title=""
+    alt=""
+    src={apmOnrampsIssuesExample}
+    style={{ maxWidth: '50%' }}
+/>
+
+If FAQ format is not applicable, use our standard headers for titles.
+
+### How do I address different tile states?
+
+A tile is something that's clickable and may contain data, depending on if customers have set it up. This is called an active vs. an empty state, and it's something to consider when writing onramps content. 
+
+  * **Empty state**: If the tile has no data, it's an empty state, and if the user clicks on it, they may be directed to a setup page. 
+    <img
+    title=""
+    alt=""
+    src={apmEmptyStateExample}
+    style={{ maxWidth: '75%' }}
+/>
+
+  * **Active state**: If the has data, it's an active state, and our onramps content will want to help users interpret the data.
+    <img
+    title=""
+    alt=""
+    src={apmActiveStateExample}
+    style={{ maxWidth: '75%' }}
+/>
+
+In the examples above, regardless of what state the tile is in, the onramp is static, so it's important to contain information that's helpful for both states. Here's the onramp content for the **Service levels** tile.
+
+ <img
+    title=""
+    alt=""
+    src={apmServiceLevelsExample}
+    style={{ maxWidth: '75%' }}
+/>
+
+
+### How long should the content be?
+
+If you're not using collapsers, **keep the character count under 1200** for the entire onramps section.
+
+### Should I bold/italicize to emphasize something?
+
+Since we can't add code formatting to onramps content, use bold and italics to emphasize something as needed.
+
+### How much detail should I include?
+
+If the onramps content isn't in the main place where the feature is (i.e., onramps content for the **Service levels** tile on the **APM Summary** page), then it’s not the place to include a lot of detail.
```

Comment:
Avoid `i.e.` and `e.g.` (a few instances of `i.e.` in this doc)

https://docs.newrelic.com/docs/style-guide/word-choice/usage-dictionary/#eg